Wed Mar 24 20:20:30 GMT 2021 : Apache Derby Network Server - 10.11.1.2 - (1629631) started and ready to accept connections on port 1527
----------------------------------------------------------------
Wed Mar 24 20:20:31 GMT 2021:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.11.1.2 - (1629631): instance a816c00e-0178-65e5-50f8-0000611cfe1c 
on database directory C:\Users\tyler\OneDrive\Documents\NetBeansProjects\RoomBookingProject\RoomManagment with class loader sun.misc.Launcher$AppClassLoader@6f94fa3e 
Loaded from file:/C:/Program%20Files/Java/jdk1.8.0_111/db/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_111-b14
user.dir=C:\Program Files\Java\jdk1.8.0_111\db
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=C:\Users\tyler\OneDrive\Documents\NetBeansProjects\RoomBookingProject
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Wed Mar 24 20:20:38 GMT 2021:
Shutting down instance a816c00e-0178-65e5-50f8-0000611cfe1c on database directory C:\Users\tyler\OneDrive\Documents\NetBeansProjects\RoomBookingProject\RoomManagment with class loader sun.misc.Launcher$AppClassLoader@6f94fa3e 
Wed Mar 24 20:20:38 GMT 2021 Thread[DRDAConnThread_2,5,main] (DATABASE = RoomManagment), (DRDAID = {2}), Database 'RoomManagment' shutdown.
Wed Mar 24 20:24:50 GMT 2021 Thread[DRDAConnThread_2,5,main] Cleanup action starting
java.sql.SQLException: Database 'RoomManagment' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
Caused by: ERROR XJ004: Database 'RoomManagment' not found.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 18 more
============= begin nested exception, level (1) ===========
ERROR XJ004: Database 'RoomManagment' not found.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.apache.derby.impl.drda.Database.makeConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(Unknown Source)
	at org.apache.derby.impl.drda.DRDAConnThread.run(Unknown Source)
============= end nested exception, level (1) ===========
Cleanup action completed
Wed Mar 24 20:24:50 GMT 2021 Thread[DRDAConnThread_2,5,main] (DATABASE = RoomManagment), (DRDAID = {3}), Database 'RoomManagment' not found.
----------------------------------------------------------------
Wed Mar 24 20:26:23 GMT 2021: Shutting down Derby engine
----------------------------------------------------------------
